src/graph.h

```diff
@@ -171,6 +172,11 @@ struct Edge {
 
   void Dump(const char* prefix="") const;
 
+  int64_t critical_time() const { return critical_time_; }
```

Comment:
Can you add a comment explaining what this value is and how it is supposed to be used. Which units too?

Reply:
Also consider explaining why this is default-initialized tp -1, this is subtle and better explained in a comment.